utils/config/nodes.hpp

```diff
@@ -153,36 +124,19 @@ class typed_leaf_node : public leaf_node {
     /// The type of the value held by this node.
     typedef ValueType value_type;
 
+    /// Constructor
```

Comment:
Missing trailing dot. But this doesn't provide a lot of information. What about something like:

`Constructs a new leaf node that contains no value.`

Reply:
Most of the other existing examples actually just say "Constructor.", so... 🤷‍♀️?